src/main/java/bowlingRefactor/Main.java

```diff
@@ -0,0 +1,27 @@
+package bowlingRefactor;
```

Comment:
```suggestion
package bowling_refactor;
```
패키지명의 카멜케이스는 컨벤션에 어긋나요. `bowling_refactor` 이나 `bowlingrefactor`로 써주세요.
(state 하위 패키지에도 같은 코멘트 드리겠습니다.)

Reply:
엇 패키지 컨벤션은 잘 몰랐는대, 이번기회에 알려주셔서 감사합니다~~